docs/sql-ref-syntax-aux-show-tables.md

```diff
@@ -18,5 +18,86 @@ license: |
   See the License for the specific language governing permissions and
   limitations under the License.
 ---
+### Description
 
-**This page is under construction**
+`SHOW TABLES` statement returns all the tables for an optionally specified `database`.
+Additionally, the output of this statement may be filtered via an optional matching
+pattern. If no database is specified then the tables are returned from the 
+current database.
+
+### Syntax
+{% highlight sql %}
+SHOW TABLES [{FROM|IN} database_name] [LIKE 'regex_pattern']
+{% endhighlight %}
+
+### Parameters
+<dl>
+  <dt><code><em>{FROM|IN} database_name</em></code></dt>
+  <dd>
+     Specifies the `database` name from which tables are listed.
+  </dd>
+  <dt><code><em>LIKE 'regex_pattern'</em></code></dt>
+  <dd>
+     Specifies the regex pattern that is used to filter out unwanted tables.
+       <br> - Only `*` and `|` are allowed as wildcard pattern.
```

Comment:
`|` isn't a wildcard pattern, so I wouldn't imply it is. 
Is it really treated as a full regex otherwise? if not, I wouldn't call it a regex unless it is generally called this in other similar references.

Reply:
@srowen Thats a very good point Sean. How about we name the pattern 'identifier_with_wildcards' (and describe as such) as mentioned in `ShowTablesCommand`'s comment ?

Reply:
Is it a SQL-like pattern that supports wildcards like `%`? then I'd just call it a `pattern`

Reply:
@srowen I quickly tried in my env. It does not support SQL-like pattern like `%`. I only saw the following work :
- `*` : Matches 0 or more characters.
- `.` : Matches a char
- `|` : works like or expression.

@shivusondur Can you please confirm ?

Reply:
@dilipbiswal @srowen 
I also tesed, it is same behaviour you mentioned above.
It will use the below method to handle, here they are handling "|" and "*" seperatly, remaining treating as regex
https://github.com/apache/spark/blob/e1ea806b3075d279b5f08a29fe4c1ad6d3c4191a/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/StringUtils.scala#L88

Reply:
OK, your comment is consistent with the docs then. Hm, but isn't it more accurate to say that `*` and `|` _don't_ work like in regexes? `*` matches 0 or more characters, and `|` delimits whole regexes, which is different, but the rest is the same. Maybe a comment more like that. "The pattern is a regex except that `*` matches 0 or more characters by itself, and `|` may only be used at the top level to delimit alternative regexes to match." ?

Reply:
@srowen @dilipbiswal 
Thank you both for your time in reviewing.

I updated according to your suggestion and screenshot also I updated.
Now it will look like below.
![image](https://user-images.githubusercontent.com/7912929/65989006-847d0280-e4a6-11e9-9b9a-3e9ebcaf92f8.png)